feedly/data.py

```diff
@@ -3,17 +3,19 @@
    handy getter methods, but otherwise you can just use a .json property to access the
    raw json passed back by the client.
 """
-from typing import Any, Dict, Optional, Callable, Union
+from typing import Any, Dict, Optional, Callable, Union, List
 from urllib.parse import quote_plus
 
 from feedly.protocol import APIClient
-from feedly.stream import STREAM_SOURCE_USER, StreamOptions, StreamBase, UserStreamId, EnterpriseStreamId, StreamIdBase, STREAM_SOURCE_ENTERPRISE
-import logging
+from feedly.stream import STREAM_SOURCE_USER, StreamOptions, StreamBase, UserStreamId, EnterpriseStreamId, StreamIdBase, \
```

Comment:
it's a bit easier to do code reviews if you don't reformat the whole file when saving. otherwise it's hard to see real diffs vs formatting changes...could you resubmit with only the code changes?

Reply:
Ah yes sorry that's Pycharm auto-reformatting at commit time 👍 
will do some time today